Generics Challenge.playground/Contents.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+
+struct CountedSet<Element: Hashable> {
+
+    private var storage: [Element: Int] = [:]
+
+    mutating func insert(_ element: Element) {
+
+        if let count = storage[element] {
+            storage[element] = count + 1
+        } else {
+            storage[element] = 1
+        }
```

Comment:
Here is a nice short hand for the same logic, I use this a lot...
```suggestion
storage[element, default: 0] += 1
```